packages/amplify-category-api/src/commands/api/import.ts

```diff
@@ -0,0 +1,66 @@
+import { $TSAny, $TSContext, AmplifyCategories, pathManager, stateManager } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import * as path from 'path';
+import fs from 'fs-extra';
+import { MySQLDataSourceAdapter, generateGraphQLSchema, Schema, Engine } from '@aws-amplify/graphql-schema-generator';
+const subcommand = 'import';
+
+export const name = subcommand;
+
+export const run = async (context: $TSContext) => {
+  const apiResourceDir = getResourceDir();
+
+  if (!apiResourceDir) {
+    throw new Error("No API Resource found.");
+  }
+
+  const config = await readConfigFile(apiResourceDir);
+
+  // TODO: Change this to a factory method once we support multiple RDS engines
+  const adapter = new MySQLDataSourceAdapter(config);
+  await adapter.initialize();
+  const models = await adapter.getModels();
+  adapter.cleanup();
+
+  const schema = new Schema(new Engine('MySQL'));
+  models.forEach(m => schema.addModel(m));
+
+  const schemaString = generateGraphQLSchema(schema);
+  writeSchemaFile(apiResourceDir, schemaString);
+
+  printer.info('Successfully imported the database schema.');
+};
+
+const getResourceDir = () => {
+  const apiNames = Object.entries(stateManager.getMeta()?.api || {})
+    .filter(([_, apiResource]) => (apiResource as $TSAny).service === 'AppSync')
+    .map(([name]) => name);
+  if (apiNames.length === 0) {
+    printer.info(
+      'No GraphQL API configured in the project.',
```

Comment:
Because this is part of `import` flow, so there might not be any existing APIs. So, we should assert length as 0?

Reply:
We need to have a GraphQL API for the import datasource to work. Since we haven't started those tasks yet, this is a temporary solution to test the import workflow with rds.env file.